### ISSUE TITLE
Screen Reader Reports That I Failed, But I Passed

### ISSUE THREAD

**Issue by jhomme** [freeCodeCamp/freeCodeCamp#37718]
**Describe your problem and how to reproduce it:**
When I go to https://www.freecodecamp.org/learn, it says I have one item in, the page says that I have 27 of 28 lessons in Basic HTML passed.  

I go to https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/create-a-set-of-checkboxes and enter the code. When I do that, the dialog comes up and says that I have passed. 
**Add a Link to the page with the problem:**

**Tell us about your browser and operating system:**
* Browser Name: Firefox
* Browser Version: 70.0.1, 64-bit
* Operating System:  Windows 10

**If possible, add a screenshot here (you can drag and drop, png, jpg, gif, etc. in this box):**
[My Code.docx](https://github.com/freeCodeCamp/freeCodeCamp/files/3825208/My.Code.docx)

**Comment by ojeytonwilliams** [comment]
I think it's the next challenge that you need to complete.  Try submitting a solution for https://www.freecodecamp.org/learn/responsive-web-design/basic-html-and-html5/use-the-value-attribute-with-radio-buttons-and-checkboxes and let me know if you've still got an uncompleted challenge.

**Comment by jhomme** [comment]
User error on my part. I was going to the wrong page.
